### DISCUSSION TITLE
Is aniseed and hotpot present the same time in the system? Would that be a problem?

### DISCUSSION THREAD

**Discussion by glyh** [nyoom-engineering/nyoom.nvim#96] [Q&A] [answered]
As title, I realize that both hotpot and aniseed(which is required by conjure)is present.

**Comment by shaunsingh** [comment, marked as answer]
>  conjure (which depends on aniseed)

The two don't conflict, so it isn't an issue. Nyoom itself is going to use hotpot to compile modules, code, run things like `nyoom repl`, etc. Aniseed is only loaded when you're actively evaluating fennel code with conjure. Even then, if you `require` a fennel file from the evaluation session that is already compiled loaded into nyoom's env (e.g. nyoom's stdlib), its going to first check if it has a cached version with hotpot, and only then is it going to use aniseed to compile

However, I do plan on adding a `hotpot` client for conjure, that uses hotpot to compile and evaluate fennel code instead of aniseed. This is far in the future though
